wandb/data_types.py

```diff
@@ -1731,8 +1733,11 @@ def numpy_arrays_to_lists(payload):
     return payload
 
 
-def val_to_json(run, key, val, step='summary'):
+def val_to_json(run, key, val, step=None):
```

Comment:
This being named `step` is really confusing since it's normally an int.  Let's call it `kind` or `namespace`.

Reply:
I kept this the same from the other three

> What are the objects since this is just an individual image?
> 
> You should definitely add a test to `tests/test_data_types.py`.
> 
> How is this different than `image-file`? Why couldn't we just extend that type with whatever you need?

It's not an individual image. It's a set of images stored individually. It's a bad name. I'm going to change it to `images/separated`